inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -9,68 +9,28 @@
 
 namespace LayerTestsUtils {
 
-LayerTestsCommon::LayerTestsCommon() : threshold(1e-2f) {
+LayerTestsCommon::LayerTestsCommon() {
```

Comment:
`LayerTestsCommon::LayerTestsCommon() :  threshold(1e-2f) {
 `

Reply:
It's syntactically impossible, because threshold field moved to parent class. To use inline initializer for parent field, it takes to declare constructor in parent class with threshold as argument.